<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QgsServerSourceSelectBase</class>
 <widget class="QDialog" name="QgsServerSourceSelectBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add Layer(s) from a Server</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="4" >
    <widget class="QGroupBox" name="GroupBox1" >
     <property name="title" >
      <string>Server Connections</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <widget class="QPushButton" name="btnNew" >
        <property name="text" >
         <string>&amp;New</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QPushButton" name="btnDelete" >
        <property name="text" >
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="btnEdit" >
        <property name="text" >
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>171</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <widget class="QPushButton" name="btnConnect" >
        <property name="text" >
         <string>C&amp;onnect</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5" >
       <widget class="QComboBox" name="cmbConnections" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="4" >
    <widget class="QGroupBox" name="btnGrpImageEncoding" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Image encoding</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4" >
    <widget class="QGroupBox" name="groupBox2" >
     <property name="title" >
      <string>Layers</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="Q3ListView" name="lstLayers" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="selectionMode" >
         <enum>Q3ListView::Multi</enum>
        </property>
        <property name="allColumnsShowFocus" >
         <bool>true</bool>
        </property>
        <property name="showSortIndicator" >
         <bool>true</bool>
        </property>
        <property name="rootIsDecorated" >
         <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Title</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Abstract</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="4" >
    <widget class="QGroupBox" name="btnGrpSrs" >
     <property name="title" >
      <string>Spatial Reference System</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButtonEspg4326" >
        <property name="text" >
         <string>ESPG:4326</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>441</width>
          <height>23</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="QPushButton" name="btnAdd" >
     <property name="text" >
      <string>&amp;Add</string>
     </property>
     <property name="shortcut" >
      <string>Alt+A</string>
     </property>
     <property name="autoDefault" >
      <bool>true</bool>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QPushButton" name="btnHelp" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Help</string>
     </property>
     <property name="shortcut" >
      <string>F1</string>
     </property>
     <property name="autoDefault" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>284</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3" >
    <widget class="QPushButton" name="btnCancel" >
     <property name="text" >
      <string>C&amp;lose</string>
     </property>
     <property name="shortcut" >
      <string>Alt+L</string>
     </property>
     <property name="autoDefault" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ListView</class>
   <extends></extends>
   <header>q3listview.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbConnections</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>btnNew</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnHelp</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
